### ISSUE TITLE
adj unwilling

### ISSUE THREAD

**Issue by arademaker** [globalwordnet/english-wordnet#177]
https://en-word.net/lemma/unwilling

(the example "an unwilling smile" is not in PWN 3.0)

How we can differentiate these two senses of unwilling?

• 02566015-a unwilling (not disposed or inclined toward; "an unwilling assistant"; "unwilling to face facts")
• 02522164-a unwilling (in spite of contrary volition)

https://www.ldoceonline.com/dictionary/unwilling suggest two syntactic patterns but the examples from 02566015-a use both.

**Comment by restinplace** [comment]
The first conflates two English meanings:  "reluctant" (but did it anyway), and "refusing" (and did not do it).  "He claimed he was an  unwilling participant in the robbery, but was unwilling to participate in his own defense."
The second means "involuntary".

**Comment by arademaker** [comment]
Hi @restinplace , the 02566015-a has seeAlso relation to 'unvoluntary'  and the 02522164-a has the similarTo relation  'unvoluntary'!  so both point to the same related concept. 

If the second means "involuntary, but not compelled.", it could be merged to http://wnpt.sl.res.ibm.com/wn/synset?id=02521353-a or 02521353-a would be the involuntary+compelled?

**Comment by arademaker** [comment]
@jmccrae I suggest rewrite the examples in https://en-word.net/lemma/unwilling to:

1. Bob gave an unwilling smile
2. Bob is an unwilling assistant / John is unwilling to face facts.

So we can make clear the syntax behaviors that differentiate the senses?

**Comment by restinplace** [comment]
Re this comment, no:
> If the second means "involuntary, but not compelled.", it could be merged ...
One sense is "not subject to one's will," the other is "despite one's contrary will."

Re example syntax, it's not that simple:
- I saw Bob's unwilling smile.
- I saw Bob's unwilling assistant.
- I saw Bob's unwilling attitude.
- She was unwilling to acknowledge her unwilling assistant's unwilling grimace.

Re rewriting examples, in this case, the meaning and implications contrast:
- Bob is an unwilling assistant / John is unwilling to face facts.
Borrowing from Agnès Varda, one sings, the other doesn't:
- Pauline is an unwilling singer / Suzanne is unwilling to sing.

In my opinion, rather than making micro changes at this point, it might be more prudent to collect canonical examples of proposed change types and consequences.
